src/components/organelle-wrapper.tsx

```diff
@@ -78,30 +78,31 @@ class OrganelleWrapper extends React.Component<OrganelleWrapperProps, OrganelleW
       height: 312
     };
 
-    modelDef.properties = modelProperties;
+    modelDef.properties = modelProperties.toJS();
 
     createModel(modelDef).then((m: any) => {
       this.model = m;
       this.completeLoad();
     });
 
-    this.handler = autorun(() => {
+    this.disposers.push(autorun(() => {
```

Comment:
Making the `modelProperties` observable allowed me to pull the autorun functions apart - I don't see any reason why they should trigger one another.